packages/git-raw-commits/index.js

```diff
@@ -65,7 +65,9 @@ function gitRawCommits (rawGitOpts, rawExecOpts) {
         transform (chunk, enc, cb) {
           isError = false
           setImmediate(() => {
-            readable.push(chunk)
+            if ( !(gitOpts?.ignore && isMatch(chunk.toString(), gitOpts?.ignore))) {
```

Comment:
Let's use regex instead of glob

```js
const ignoreRegex = typeof gitOpts.ignore === 'string'
  ? new RegExp(gitOpts.ignore)
  : gitOpts.ignore
const shouldNotIgnore = ignoreRegex
  ? chunk => !ignoreRegex.test(chunk.toString())
  : () => true
...
if (shouldNotIgnore(chunk)) {
  readable.push(chunk)
}
```